docs/troubleshooting.md

```diff
@@ -0,0 +1,13 @@
+# Troubleshooting
+
+## "Undetermined" State Interactive Apps
```

Comment:
Nicely written -- describes the problem and solution at a high level without getting too much in the weeds.